### ISSUE TITLE
`make start` fails on macOS due to missing npm

### ISSUE THREAD

**Issue by munnerz** [cert-manager/website#1570]
I think we need to include download links/info for npm on darwin/arm64:

```
~/go/src/github.com/cert-manager/website git:[fixup-logo]
make start
make: *** No rule to make target `/Users/james/go/src/github.com/cert-manager/website/_bin/downloaded/tools/npm@v20.11.1_darwin_arm64', needed by `_bin/tools/npm'.  Stop.
```

**Comment by munnerz** [comment]
🤦 probably because I didn't install npm which is a pre-req of this repository...

**Comment by maelvls** [comment]
Your problem was real! The PR https://github.com/cert-manager/makefile-modules/pull/124/files#r1950881680 broke the API of `ADDITIONAL_TOOLS`, which in turn broke `make check`. I've proposed a fix in #1644.
